stacktracer/src/main/scala-2/zio/internal/stacktracer/Tracer.scala

```diff
@@ -28,6 +28,6 @@ object Tracer {
 sealed trait Tracer {
   type Type <: AnyRef
   val empty: Type with Tracer.Traced
-  def unapply(trace: Type): Option[(String, String, Int)]
```

Comment:
We need to keep this for bin-compat reasons. While it's unlikely that someone will call this method directly, it might exist in compiled code due to macro inlining etc.

Reply:
Done